step/src/main/webapp/prefForm.jsp

```diff
@@ -36,47 +36,43 @@ limitations under the License.
     
     
     <% if (!userService.isUserLoggedIn()) {
-        response.sendRedirect("index.jsp");   
+        response.sendRedirect("/index.jsp");   
     } %>
     <section class="prefform">
-    <form id="regForm" action="Pref" method="POST"> 
+    <form id="regForm" action="Pref" class="inputGroup" method="POST"> 
         <div class="tab"> <div class="question"> Are you staying in the US now? </div>
-
             <p>
-                <input type="radio" oninput="this.className = ''" name="q1" value="Yes" id="q1yes">
+                <input type="radio" oninput="this.className = ''" name="q1" value="Yes" id="q1yes" required>
                 <label for="q1yes"> Yes </label>
             </p>
             <p>
                 <input type="radio" oninput="this.className = ''" name="q1" value="No" id="q1no">
                 <label for="q1no"> No </label>
             </p>   
-        </div>
-        <br>  
+        </div> 
         <div class="tab"> <div class="question"> Do you have any pets? </div>
             <p>
-                <input type="radio" oninput="this.className = ''" name="q2" value="Yes" id="q2yes">
+                <input type="radio" oninput="this.className = ''" name="q2" value="Yes" id="q2yes" required>
```

Comment:
Are we making questionnaire answers required now? Fine by me

Reply:
No I planned to put another variable "not willing to answer" at first and set it's required. However, it did not work. I deleted it